src/DaiDripsHub.sol

```diff
@@ -24,7 +24,7 @@ contract DaiDripsHub is ERC20DripsHub {
         uint64 cycleSecs,
         IDai _dai,
         IERC20Reserve reserve
-    ) ERC20DripsHub(cycleSecs, _dai, reserve) {
+    ) ERC20DripsHub(cycleSecs, reserve) {
```

Comment:
Maybe we should support the general `permit` standard as well.

Reply:
I agree, that will be the responsibility of the drivers